weave/integrations/litellm/litellm_test.py

```diff
@@ -146,7 +146,7 @@ def test_litellm_quickstart_stream(
         if chunk.choices[0].delta.content:
             all_content += chunk.choices[0].delta.content
 
-    exp = """Hello! I'm just a virtual assistant so I don't have feelings, but I'm here and ready to help you with anything you need. How can I assist you today?"""
```

Comment:
why did the litellm tests change? seems like you can just revert all changes w.r.t lite llm, no?

Reply:
yeah I can change it back. They changed because I ran the test in record mode which changed the response slightly, hence the update. but I can revert it back.

Reply:
commented above.